### PR TITLE
Remove popover after the tag is deleted

### LINKED ISSUE #5342: Tag popovers don't disappear when tags are deleted
Follow up on #5072 -- when you delete a tag, we need to hide the popover first!

image

I'd love some help with this! 

https://github.com/publiclab/plots2/blob/master/app/views/tag/_tagging.html.erb

### PR DESCRIPTION
**Pull request by CleverFool77**
Fixes #5342 (<=== Add issue number here)

## Earlier
beforepr

## After Changes
afterpr

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

Thanks!

### CODE REVIEW

**Comment by CleverFool77** [comment]
@publiclab/plots2-reviewers 
Thanks !!!

**Comment by CleverFool77** [comment]
I found the bug in this, I'll fix it till evening.

File: app/views/tag/_tags.html.erb [hunk 1]
```diff
@@ -52,3 +52,11 @@ $(".pop").popover({ trigger: "manual" , html: true, animation:false})
...
 
 </script>
+
+<script>
+$('.tag-delete').click(
+  function(){
+    $('.popover').remove();
+  }
+);
+</script>
```
Comment (gauravano) [inline app/views/tag/_tags.html.erb]: Although, the code looks good. Can we add the code in the above <script> tag? Thanks!

Comment (CleverFool77) [reply app/views/tag/_tags.html.erb]: Sure @gauravano  !!!

**Comment by CleverFool77** [comment]
Hi @gauravano ,
I've updated the PR as requested.

**Comment by gauravano** [comment]
Merged :100: :tada:

**Pull request merged**